buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersionHelper.kt

```diff
@@ -41,6 +41,11 @@ object IntellijVersionHelper {
     return version.substring(0, 6)
   }
 
+  fun Properties.getEapOfLatestSupportedMajor(): String? {
+    val maybeEap = getProperty("eapOfLatestSupportedMajor")
+    return if (maybeEap == "") null else maybeEap
+  }
```

Comment:
`getEapOfLatestSupportedMajor` is too case-specific to be added as an extension function to `Property`... I'd rather do sth like:

```suggestion
  fun Properties.getPropertyOrNullIfEmpty(key: String): String? {
    val value = getProperty(key)
    return if (value == "") null else value
  }
```